keyless-entry

```diff
@@ -37,7 +37,12 @@
 # accessible on boot and executable or it won't work.
 #
 # You will be prompted to enter an existing LUKS key (passphrase)
-# once for each encrypted device during configure.
+# once for each encrypted device during configure. This will add a 
+# master key for each should be stored on an encrypted partition to allow
```

Comment:
There's something wrong in your language here in this added comment. Please clean up the language.

Reply:
I've pushed an update to fix the language in this comment and to pass flake8.